kotlin-analysis-api/src/test/kotlin/com/google/devtools/ksp/test/KSPAATest.kt

```diff
@@ -146,6 +146,18 @@ class KSPAATest : AbstractKSPAATest() {
         runTest("../test-utils/testData/api/builtInTypes.kt")
     }
 
+    @TestMetadata("ObjCacheA.kt")
+    @Test
+    fun testObjCacheA() {
```

Comment:
name suggests objCache, I assume it means the caches in psi manager as suggested by the code above rather than the KSP's own object cache?

Reply:
No, it is KSP's object cache. PSI and AA caches need to be cleared too.